docs/sql-ref-ansi-compliance.md

```diff
@@ -69,19 +69,19 @@ The type conversion of Spark ANSI mode follows the syntax rules of section 6.13
 
  The valid combinations of target data type and source data type in a `CAST` expression are given by the following table.
 “Y” indicates that the combination is syntactically valid without restriction and “N” indicates that the combination is not valid.
-    
-| From\To   | NumericType | StringType | DateType | TimestampType | IntervalType | BooleanType | BinaryType | ArrayType | MapType | StructType |
+
+| Source\Target | Numeric | String | Date | Timestamp | Interval | Boolean | Binary | Array | Map | Struct |
```

Comment:
We could even make it lower-cased and say these are simple strings or catalog strings .. but I think it's fine either way.

Reply:
Considering there are "Y"s and "N"s in the table, let's keep the current way.

Reply:
👌